--- lz4/setup.py	2017-10-11 14:46:21.328659517 -0400
+++ lz4-patched/setup.py	2017-10-11 14:47:51.818295462 -0400
@@ -41,11 +41,11 @@
 
 # Check to see if we have a lz4 library installed on the system and
 # use it if so. If not, we'll use the bundled library.
-liblz4_found = library_is_installed('liblz4')
+liblz4_found = False#library_is_installed('liblz4')
 
 # Check to see if we have the py3c headers installed on the system and
 # use it if so. If not, we'll use the bundled library.
-py3c_found = library_is_installed('py3c')
+py3c_found = False#library_is_installed('py3c')
 
 # Set up the extension modules. If a system wide lz4 library is found, we'll
 # use that. Otherwise we'll build with the bundled one. If we're building
@@ -153,7 +153,7 @@
     use_scm_version={
         'write_to': "lz4/version.py",
     },
-    setup_requires=['setuptools_scm'],
+    #setup_requires=['setuptools_scm'],
     description="LZ4 Bindings for Python",
     long_description=open('README.rst', 'r').read(),
     author='Jonathan Underwood',
